src/server.rs

```diff
@@ -1,22 +1,24 @@
-use std::borrow::Cow;
-use std::collections::{hash_map::Entry, HashMap};
-use std::sync::{Arc, Mutex};
+use std::{
```

Comment:
Curious about this (no change needed, necessarily). I find the former easier to read. Was this a formatter change, or did you change this manually? If you changed it manually, do find this style easier to read?